spec/controllers/renewals_controller_spec.rb

```diff
@@ -86,6 +94,16 @@
       end
     end
 
+    context 'when response include errored items with empty error message' do
+      let(:renew_items_response) { { success: [checkouts[0]], error: [[checkouts[2], '']] } }
+
+      it 'error messages include items title only if non renewal reason empty' do
```

Comment:
```suggestion
      it 'error messages include items title only' do
```
Seems like `if non renewal reason empty` is redundant?

Reply:
It happens when we get the java errors so I am not adding a detailed msg for the renewal flash, so there should be a div with text starting with `Denied:` in this case, that is what I was trying to test here but not sure not required to do so 🤔

Reply:
So, line 97 says `items with empty error message`

I thought that was the same thing?

Reply:
ahh I see, yes seems redundant